walkthroughs/week-5-tdd/project/src/test/java/com/google/sps/FindMeetingQueryTest.java

```diff
@@ -270,5 +270,40 @@ public void notEnoughRoom() {
 
     Assert.assertEquals(expected, actual);
   }
+
+  @Test(expected =  IllegalArgumentException.class)
+  public void slotBelowRange() {
+    // Since slot < 0, we expect an IllegalArgumentException to be thrown.
+
+    query.convertToMinuteTimestamp(-1);
+  }
+
+  @Test(expected =  IllegalArgumentException.class)
+  public void slotAboveRange() {
+    // Since slot > 47, we expect an IllegalArgumentException to be thrown.
+
+    query.convertToMinuteTimestamp(48);
+  }
+
+  @Test
+  public void slotOnBoundary() {
+    // The result should be the number of minutes elapsed up to the last half hour of the day.
+    int slot = 47;
+
+    int actual = query.convertToMinuteTimestamp(slot);
+    int expected = 1410;
```

Comment:
Maybe not needed here, but it's nice to add references/short comments to where such constants are coming from and why exactly that is expected.

Reply:
Seen. Thanks.